### PR TITLE
Frax Finance Fraxtal [frax-finance-fraxtal]

### PR DESCRIPTION
**Pull request by FortisFortuna**
Changes proposed in this pull request:
Add support for Frax Finance voting on the Fraxtal chain

### CODE REVIEW

File: src/strategies/frax-finance-fraxtal/index.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  },
+];
```
Comment (ChaituVR) [inline src/strategies/frax-finance-fraxtal/index.ts:36]:
```suggestion
const abi = [ 'function balanceOf(address account) view returns (uint256)' ]
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/frax-finance-fraxtal/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+import { multicall } from '../../utils';
+
+const BIG18 = BigNumber.from('1000000000000000000');
```
Comment (ChaituVR) [inline src/strategies/frax-finance-fraxtal/index.ts:5]:
```suggestion
// const BIG18 = BigNumber.from('1000000000000000000');
```

File: src/strategies/frax-finance-fraxtal/index.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
```
Comment (ChaituVR) [inline src/strategies/frax-finance-fraxtal/index.ts:2]:
```suggestion
// import { BigNumber } from '@ethersproject/bignumber';
```

**Review by ChaituVR — APPROVED** [review]

**Comment by pegahcarter** [comment]
Big moves here!

**Pull request merged**